packages/flutter/test/widget/scrollable_list_horizontal_test.dart

```diff
@@ -27,9 +28,10 @@ Widget buildFrame() {
 }
 
 void main() {
-  test('Drag horizontally', () {
+  test('Drag horizontally with scroll anchor at top', () {
     testWidgets((WidgetTester tester) {
-      tester.pumpWidget(buildFrame());
+      tester.pumpWidget(new Container());
```

Comment:
you shouldn't need this. pumpWidget() pumps an ErrorWidget between each pump just to clear the air.